rosidl_cmake/cmake/rosidl_export_typesupport_targets.cmake

```diff
@@ -15,13 +15,15 @@
 #
 # Export typesupport targets to downstream packages.
 #
+# :param library_suffix: the suffix of the library
+# :type library_suffix: string
 # :param ARGN: a list of targets.
 #   Each element must be an exported CMake library target.
 # :type ARGN: list of strings
 #
 # @public
 #
-macro(rosidl_export_typesupport_targets)
+macro(rosidl_export_typesupport_targets library_suffix)
```

Comment:
This is an API breaking change. Why should we land that after the API freeze?

Reply:
The function is supposed to append the passed targets to a `@PROJECT_NAME@_TARGETS${_suffix}` variable.
Instead of that, it's appending it to `@PROJECT_NAME@_TARGETS`, as `${_suffix}` is always empty (if it's not empty, it's a value randomly set previously by another thing in the same scope).

I didn't see a way of fixing behavior without modifying the API, and thus I modified it.

Reply:
It's unfortunate indeed, but I don't see a way around it either.